### PR TITLE
Added request/dependency telemetry for actors.

### PR DESCRIPTION
**Pull request by arturek**
Added SF dependency telemetry to API project.

### CODE REVIEW

File: src/Bullfrog.Actors/MonitoredActorService.cs [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            }
+
+            public void SetParameter(int position, string parameName, object parameter)
+            {
+                parameters[parameName] = parameter;
+            }
+
+            public object GetParameter(int position, string parameName, Type paramType)
+            {
+                return parameters[parameName];
```
Comment (hudo) [inline src/Bullfrog.Actors/MonitoredActorService.cs]: is it ok to throw an exception here if param doesn't exists?

Comment (arturek) [reply src/Bullfrog.Actors/MonitoredActorService.cs]: It's based on the original code. Unfortunately it's internal so I need to add my own version.

Comment (dguerin) [reply src/Bullfrog.Actors/MonitoredActorService.cs]: If it's not there I would expect it to blow up anyway

File: src/Bullfrog.Actors/MonitoredActorService.cs [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+using Microsoft.ServiceFabric.Actors.Remoting.V2.Runtime;
+using Microsoft.ServiceFabric.Actors.Runtime;
+using Microsoft.ServiceFabric.Services.Communication.Runtime;
+using Microsoft.ServiceFabric.Services.Remoting.V2;
+using Microsoft.ServiceFabric.Services.Remoting.V2.Runtime;
+
+namespace Bullfrog.Actors
+{
+    /// <summary>
+    /// Creates actors with telemetry components injected into remoting pipeline to track requests and dependecies.
```
Comment (VaclavK) [inline src/Bullfrog.Actors/MonitoredActorService.cs]: typo

**Pull request merged**